packages/svelte/src/compiler/migrate/index.js

```diff
@@ -27,9 +27,10 @@ const style_placeholder = '/*$$__STYLE_CONTENT__$$*/';
  * May throw an error if the code is too complex to migrate automatically.
  *
  * @param {string} source
+ * @param {string} [filename]
  * @returns {{ code: string; }}
  */
-export function migrate(source) {
+export function migrate(source, filename) {
```

Comment:
let's play it save and make this an options argument, so we can enhance this in the future of needed

Reply:
Good idea, will do 😉